cli/trigger.go

```diff
@@ -7,7 +7,6 @@ import (
 
 	log "github.com/Sirupsen/logrus"
 	"github.com/bitrise-io/bitrise/models"
-	"github.com/bitrise-io/bitrise/version"
```

Comment:
why is this line removed, but all others are commented?

Reply:
Version is only used in `PrintBitriseHeaderASCIIArt(version.VERSION)`. Since this method is commented the import is not needed anymore, However, if we uncomment the method, the VScode Go plugin most likely will add this line automatically.